lib/linguist/languages.yml

```diff
@@ -5328,6 +5328,19 @@ PowerShell:
   interpreters:
   - pwsh
   language_id: 293
+Praat:
+  type: programming
+  color: "#c8506d"
+  tm_scope: source.praat
+  ace_mode: praat
+  extensions:
+  - ".praat"
+  - ".praat-script"
+  - ".praat_script"
+  - ".praatscript"
+  interpreters:
+  - praat
```

Comment:
We only need the interpreter if the language is commonly used in shell scripts and this has a hashbang line at the very beginning.

Reply:
I see, I removed that now.

To be honest: I was under the impression that the shebang was supported but simply underused (Most users are on Windows, and double-click the file in explorer), but, after testing, I can confirm it doesn't work with the current latest version of Praat.